en/docs/basics_topics/distributed_train.md

```diff
@@ -58,43 +58,43 @@ if __name__ == '__main__':
 
 ### Configuration of ports and device
 
-In  `oneflow.config` , we provide distributed related port. We mainly use two of them:
+In  `oneflow.config` , we provide distributed program related interface. We mainly use two of them:
```

Comment:
we provide distributed program related interface
->
we provide the interfaces  related to distributed program